test/lib/ufe/testComboCmd.py

```diff
@@ -805,6 +805,55 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
+    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 124, 'Center point command is only available in Maya Preview Release 124 or later.')
```

Comment:
@ppt-adsk  for now I set the PR version to 124.  We might want to change this or come up with another way to detect my changes on Maya side.